Sources/ApolloTestSupport/MockGraphQLServer.swift

```diff
@@ -40,7 +40,7 @@ public class MockGraphQLServer {
     }
   }
   
-  public typealias RequestHandler<Operation: GraphQLOperation> = (HTTPRequest<Operation>) -> JSONObject
+  public typealias RequestHandler<Operation: GraphQLOperation> = (HTTPRequest<Operation>) -> Result<JSONObject, Error>
```

Comment:
I'm not sure we need to return `Result` here. As the handler is synchronous, it might be easier to make it throwing instead.